README.md

```diff
@@ -30,3 +30,23 @@ $ ./out/release/x64/walrus test.wasm // run walrus executable
 
 This will produce build files using CMake's default build generator. Read the
 CMake documentation for more information.
+
+## Perf
+
+You'll need [Perf](https://perf.wiki.kernel.org/index.php/Main_Page).
+
+0. To compile with perf support, use `-DWALRUS_JITPERF=1`
+
+1. Run Walrus with: `sudo perf record -k 1 walrus --jit --perf WALRUS_PARAMETERS`
+
+    It'll generates 2 files:
+   - perf.data - it is generated by perf
+   - jit-XXXXXX.dump - JITDump file, XXXXXX is the PID of process, and it contains details of jitted functions
+
+2. After that you should chown these 2 files
+
+3. Inject JITDump file into `perf.data` with `perf inject --jit -i perf.data -o perf.data.jitted`
+
+    It'll generate many shared object files, and `perf.data.jitted`
```

Comment:
Add a comment that it is recommended to do in a temporary sub-directory

Reply:
No, because the shared objects will be generated to source folder, and you couldn't move it, because perf report won't work.